easybuild/easyconfigs/s/staden/staden-2.0.0b11-foss-2015b.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'staden'
+version = '2.0.0b11'
+
+homepage = 'http://sourceforge.net/projects/staden/files/staden/'
+description = """A fully developed set of DNA sequence assembly (Gap4 and Gap5), editing and analysis tools (Spin) for
+ Unix, Linux, MacOSX and MS Windows."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+source_urls = [('http://sourceforge.net/projects/staden/files/%(namelower)s/%(version)s', 'download')]
+sources = ['%(namelower)s-%(version)s-2016-src.tar.gz']
+
+dependencies = [
+    ('io_lib', '1.14.9'),
+    ('Tcl', '8.6.5'),
+    ('Tk', '8.6.5'),
+    ('tklib', '0.6'),
+    ('zlib', '1.2.8'),
+    ('libpng', '1.6.21'),
+    ('cURL', '7.45.0'),
+]
+
+configopts = ' --with-tcl=$EBROOTTCL/lib'
+configopts += ' --with-tk=$EBROOTTK/lib'
+configopts += ' --with-tklib=$EBROOTTKLIB/lib/tklib0.6'
```

Comment:
This can all go on a single line imho, but we should try to avoid hardcoding the `tklib` version.

This can probably be done by using `.../lib/tklib*` here?